R/internal.R

```diff
@@ -1,1474 +1,1500 @@
+        } else if (class(override.se) != "list") {
+            warning("SEs must be provided as a list. Using default SEs.")
+            se <- models[[i]]@se
+        } else if (length(override.se) != length(models)) {
+            warning(paste("Number of SEs provided does not match number of models.", 
+                          "Using default SEs."))
+            se <- models[[i]]@se
+        } else if (length(models[[i]]@se) != length(override.se[[i]])) {
+            warning(paste("Number of SEs provided does not match number of ", 
+                          "coefficients in model ", i, ". Using default SEs.", sep = ""))
+            se <- models[[i]]@se
+        } else if (class(override.se[[i]]) != "numeric") {
+            warning(paste("SEs provided for model", i, 
+                          "are not numeric. Using default SEs."))
+            se <- models[[i]]@se
+        } else {
+            se <- override.se[[i]]
+        }
+        models[[i]]@se <- se
+        
+        # p values
+        if (class(override.pval) != "list" && length(override.pval) == 1 && 
+            override.pval == 0) {
+            pval <- models[[i]]@pvalues
+        } else if (class(override.pval) == "numeric" && length(models) == 1 && 
+                   length(override.pval) == length(models[[i]]@pvalues)) {
+            pval <- override.pval
+        } else if (class(override.pval) != "list") {
+            warning("p values must be provided as a list. Using default p values.")
+            pval <- models[[i]]@pvalues
+        } else if (length(override.pval) != length(models)) {
+            warning(paste("Number of p values provided does not match number of", 
+                          "models. Using default p values."))
+            pval <- models[[i]]@pvalues
+        } else if (length(models[[i]]@se) != length(override.pval[[i]])) {
+            # previous line: comparison with se because pvalues can be empty
+            warning(paste("Number of p values provided does not match number of ", 
+                          "coefficients in model ", i, ". Using default p values.", sep = ""))
+            pval <- models[[i]]@pvalues
+        } else if (class(override.pval[[i]]) != "numeric") {
+            warning(paste("p values provided for model", i, 
+                          "are not numeric. Using default p values."))
+            pval <- models[[i]]@pvalues
+        } else {
+            pval <- override.pval[[i]]
+        }
+        models[[i]]@pvalues <- pval
+        
+        # lower bound of confidence intervals
+        if (is.null(override.ci.low)) {
+            # do nothing
+        } else if (class(override.ci.low) != "list" && length(override.ci.low) 
+                   == 1 && override.ci.low == 0) {
+            ci.low <- models[[i]]@ci.low
+        } else if (class(override.ci.low) == "numeric" && length(models) == 1 && 
+                   length(override.ci.low) == length(models[[i]]@coef)) {
+            ci.low <- 
```

Comment:
Can you fix the indentation here? I would either just get rid of the line break or have the three arguments of `gsub` aligned under each other in three separate lines.